config/software/make.rb

```diff
@@ -35,9 +33,6 @@
 
   # Work around an error caused by Glibc 2.27
```

Comment:
We should remove these comments as well.

Reply:
Removed the unnecessary comments